crates/dojo-world/src/contracts/model.rs

```diff
@@ -146,6 +147,10 @@ impl<'a, P> ModelReader<ModelError> for ModelRPCReader<'a, P>
 where
     P: Provider + Sync + Send,
 {
+    fn name(&self) -> String {
+        self.name.to_string()
+    }
```

Comment:
We may actually add the `selector()` function right away, and compute the selector from the name for now? And later, it will simply return the `self.selector`, WDYT?

Reply:
really good idea!